### PR TITLE
add Indonesian language

### PR DESCRIPTION
**Pull request by GarudaID**

### CODE REVIEW

**Comment by GarudaID** [comment]
I added some Indonesian language in the Readme file

File: translations/README.id.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+# Mulai
+
+> **Guru**, kita punya [termasuk beberapa saran](for-teachers.md) tentang bagaimana menggunakan kurikulum ini. Kami akan menyukai tanggapan Anda [di forum diskusi kami](https://github.com/microsoft/Web-Dev-For-Beginners/discussions/categories/teacher-corner)!
+> **Pelajar**, untuk menggunakan kurikulum ini sendiri, bagilah seluruh repo dan selesaikan latihannya sendiri, dimulai dengan kuis pra-kuliah, kemudian membaca kuliah dan menyelesaikan sisa kegiatan. Cobalah untuk membuat proyek dengan memahami pelajaran daripada menyalin kode solusi; namun kode tersebut tersedia di folder /solutions di setiap pelajaran berorientasi proyek. Ide lain adalah membentuk kelompok belajar dengan teman-teman dan membahas isinya bersama-sama. Untuk studi lebih lanjut, kami merekomendasikan [Belajar Microsoft](https://docs.microsoft.com/users/jenlooper-2911/collections/jg2gax8pzd6o81?WT.mc_id=academic-13441-cxa) dan dengan menonton video yang disebutkan di bawah ini.
+[![Promo video](images/web.gif)](https://youtube.com/watch?v=R1wrdtmBSII "Promo video")
```
Comment (jlooper) [inline translations/README.id.md:23]: hi, the images will need relative links - ../../

**Review by jlooper — CHANGES_REQUESTED** [review]

**Comment by GarudaID** [comment]
Fix project images links in README.md

**Comment by jlooper** [comment]
hi, the links to the lessons need to be fixed (the ones in the table) - add '../' as necessary to avoid 404s

File: translations/README.id.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+# Mulai
+
+> **Guru**, kita punya [termasuk beberapa saran](for-teachers.md) tentang bagaimana menggunakan kurikulum ini. Kami akan menyukai tanggapan Anda [di forum diskusi kami](https://github.com/microsoft/Web-Dev-For-Beginners/discussions/categories/teacher-corner)!
+> **Pelajar**, untuk menggunakan kurikulum ini sendiri, bagilah seluruh repo dan selesaikan latihannya sendiri, dimulai dengan kuis pra-kuliah, kemudian membaca kuliah dan menyelesaikan sisa kegiatan. Cobalah untuk membuat proyek dengan memahami pelajaran daripada menyalin kode solusi; namun kode tersebut tersedia di folder /solutions di setiap pelajaran berorientasi proyek. Ide lain adalah membentuk kelompok belajar dengan teman-teman dan membahas isinya bersama-sama. Untuk studi lebih lanjut, kami merekomendasikan [Belajar Microsoft](https://docs.microsoft.com/users/jenlooper-2911/collections/jg2gax8pzd6o81?WT.mc_id=academic-13441-cxa) dan dengan menonton video yang disebutkan di bawah ini.
+
+[![Promo video](https://github.com/GarudaID/Web-Dev-For-Beginners/blob/main/images/web.gif)](https://youtube.com/watch?v=R1wrdtmBSII "Promo video")
```
Comment (jlooper) [inline translations/README.id.md:24]: Wrong link

**Review by jlooper — CHANGES_REQUESTED** [review]

**Pull request merged**